db/products.js

```diff
@@ -0,0 +1,72 @@
+const {client} = require('./index');
+	console.log('CLIENT', client);
```

Comment:
REQUIRED - Remove console.logs that aren't used for output that we want to see all the time (i.e. remove debug console.logs)